### PR TITLE
Fullscreen mode+

### PR DESCRIPTION
**Pull request by quardbreak**
* Adds option "Full" which will hide menu and statusbar in fullscreen mode
* Adds chat input position setting, which can alter input field under output text

image

### CODE REVIEW

**Review by MistakeNot4892 — APPROVED** [review]
Willing to give this a shot. Fuck me if that skin code isn't ugly as Hell though.

**Review by deathride58 — APPROVED** [review]
[Not hiding the verb menu? You are tiny grasshopper.](https://i.redd.it/s6j7q7eefw941.png) (/s)

Jokes aside though, the code looks alright, even with it being a little messy. However, you might actually be able to make use of splitters here to make this a lot more elegant. You can do this by turning the command bar into it's own control, adding a horizontal splitter layer to both the main control and the chat control, then have the code side of things simply give the command bar control to the appropriate bottom splitter (and of course, scale the splitter so that the command bar retains the proper size). This would remove the need for having completely separate controls for both layouts, which would reduce maintenance overhead for things that require touching interface controls directly (for example, if we wanted to make the command bar able to show typing indicators, or wanted to bring back the classic "GLORF" when you're hit with a strong weapon while typing)

I'll be giving this 24 hours from this comment being posted before I merge it, in case any other maintainers wants to make my idea detailed above a requested change, or in case you want to rework it yourself regardless.

**Pull request merged**
